controllers/item.js

```diff
@@ -19,11 +19,23 @@ item.withFields = (queryBuilder) => {
   // Category
     .leftJoin('category', 'item.categoryID', 'category.categoryID')
     .select('category.name as category')
+
+  // Mapping between query param fields and database query column names
+  const filterParams = {
+    brand: 'brand.brandID',
+    model: 'model.modelID',
+    category: 'category.categoryID'
+  }
```

Comment:
I don't think that'd be necessary. I'll always have the ids since I have to get the brands, models and categories to fill the filtering options anyway. And comparing with ids seem better than doing it with strings. So I don't think it is worth implementing for now at least.